### PR TITLE
Feat: Added Event Submission Module

### PR DESCRIPTION
**Pull request by HowardMak**
Added Event Submission Module:
- Added Event server module
- Fixed query and mutation bugs
- Internationalised the Event module

Resolve: CP-18

## TL;DR

- Implement authLink and responseLink in the apolloClient.ts to add access token and intercept auth status and new access token to localStorage

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Refactoring (no functional changes, only code improvements)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Files for Review

1. `src/apollo/apolloClient.ts`

## Resolve
Resolve CP-1

### CODE REVIEW

File: src/modules/event/event.resolver.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    @Query(() => [EventDto])
+    async findEventsBySubmitter(@Args('submitter') submitter: string): Promise<EventDto[]> {
+        return await this.eventService.findEventsBySubmitter(submitter);
+    }
+
+    @Query(() => [EventDto])
+    async findEventsByOrgName(@Args('orgName') orgName: string): Promise<EventDto[]> {
+        return await this.eventService.findEventsByOrgName(orgName);
+    }
```
Comment (danieltate888) [inline src/modules/event/event.resolver.ts:35]: For filter events, you should have a thinking about how can we implement it. filter at backend or frontend.

By the way, this is just a reminder.
